packages/dinero.js/src/api/__tests__/toFormat.test.ts

```diff
@@ -11,58 +12,164 @@ import {
 } from '../../../../../test/utils';
 
 describe('toFormat', () => {
+  const simpleFormatter = ({ amount, currency }) =>
+    `${currency.code} ${amount}`;
+
   describe('number', () => {
     const dinero = createNumberDinero;
 
-    it('formats the Dinero object with the passed transformer', () => {
-      const formatter = ({ amount, currency }) => `${currency.code} ${amount}`;
-      const d = dinero({ amount: 500, currency: USD });
+    describe('with simple formatter', () => {
+      it('formats the Dinero object with the passed transformer', () => {
+        const d = dinero({ amount: 500, currency: USD });
+
+        expect(toFormat(d, simpleFormatter)).toBe('USD 5');
+      });
+      it('formats the Dinero object with the passed transformer using the scale', () => {
+        const d = dinero({ amount: 4545, currency: USD, scale: 3 });
 
-      expect(toFormat(d, formatter)).toBe('USD 5');
+        expect(toFormat(d, simpleFormatter)).toBe('USD 4.545');
+      });
     });
-    it('formats the Dinero object with the passed transformer using the scale', () => {
-      const formatter = ({ amount, currency }) => `${currency.code} ${amount}`;
-      const d = dinero({ amount: 4545, currency: USD, scale: 3 });
+    describe('with complex formatter', () => {
+      const complexFormatter: Transformer<
```

Comment:
You don't need to type the formatter, type inference is enough here. This should allow you to hoist this instead of recreating it in the `bigint` and Big.js tests.